common/persistence/nosql/nosqlplugin/cassandra/workflow_parsing_utils_test.go

```diff
@@ -0,0 +1,233 @@
+// The MIT License (MIT)
+
+// Copyright (c) 2017-2020 Uber Technologies Inc.
+
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+package cassandra
+
+import (
+	"testing"
+	"time"
+
+	cql "github.com/gocql/gocql"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/uber/cadence/common"
+	"github.com/uber/cadence/common/persistence"
+)
+
+type mockUUID struct {
+	uuid string
+}
+
+func (m mockUUID) String() string {
+	return m.uuid
+}
+
+func Test_parseWorkflowExecutionInfo(t *testing.T) {
+
+	completionEventData := []byte("completion event data")
+	autoResetPointsData := []byte("auto reset points data")
+	searchAttributes := map[string][]byte{"AttributeKey": []byte("AttributeValue")}
+	memo := map[string][]byte{"MemoKey": []byte("MemoValue")}
+	partitionConfig := map[string]string{"PartitionKey": "PartitionValue"}
+	timeNow := time.Now()
+
+	tests := []struct {
+		args map[string]interface{}
+		want *persistence.InternalWorkflowExecutionInfo
+	}{
+		{
+			args: map[string]interface{}{
+				"domain_id":                             mockUUID{"domain_id"},
```

Comment:
nit: Most of the style guides and some linters stand against struct initialization.

See more https://github.com/uber-go/guide/blob/master/style.md#initializing-structs

I understand that, in this case, you won't extend the struct with extra fields, but I suggest adding a function that creates a mockUUID to avoid confusion.

Reply:
Thanks, i have added a private function to initialize the mockUUID.